nienaruszalnosc-danych.rst

```diff
@@ -139,6 +138,19 @@ System KSI pozwala na udowodnienie, że określony stan bazy danych istniał w k
 2. Odwoływanie się do łańcucha sum kontrolnych w podpisach elektronicznych. Dzięki temu podpis uczciwego człowieka składany na dokumencie w danej sprawie poświadczałby również historię tej sprawy i uniemożliwiał jej zmianę do tego punktu.
 3. Publikację cząstkowych sum kontrolnych w inny sposób (np. publikacja sumy konrolnej sprawy w powiadomieniach emailowych wysyłanych obywatelowi).
 
+Długoterminowa archiwizacja podpisów
+------------------------------------
+
+Tradycyjna metoda konserwacji podpisu polega na opatrywaniu go pieczęcią elektroniczną poświadczającą, że podpis został utworzony przed określonym punktem czasowym (w którym tylko właściwa osoba dysponowała danymi pozwalającymi na złożenie tego podpisu). Pieczęć korzysta przy tym z tej samej technologii, co sam podpis. Zwiększenie mocy obliczeniowej komputerów niesie ze sobą ryzyko złamania zarówno jednego, jak i drugiego. Przyjmuje się, że wejście w życie komputerów kwantowych będzie oznaczało koniec współcześnie popularnych metod podpisywania (por. dokument “`CNSA Suite and Quantum Computing FAQ`_”). Za pomocą komputerów kwantowych będzie można, mając dostęp do publicznych certyfikatów, podrobić zarówno podpis pod dokumentem, jak i pieczęć znakującą ten podpis czasem.
+
+Konserwacja podpisów w oparciu o Keyless Signatures’ Infrastructure bazuje tymczasem nie na tej samej technologii, co podpis (kryptografia klucza publicznego), tylko na funkcjach skrótu, które są uznawane za odporne na zagrożenia związane z informatyką kwantową. Co za tym idzie, o ile ktoś byłby w stanie w przyszłości podrobić podpis stosowany współcześnie, to nie będzie w stanie udowodnić, że podpis ten został współcześnie stworzony, co jest kluczowe przy jego weryfikacji (certyfikaty wykorzystywane do tworzenia podpisów mają swoje okresy ważności).
```

Comment:
Nie można powiedzieć, że danych poświadczonych w KSI nigdy nie będzie się dało podrobić.

Być może informatyka kwantowa nie stanowi zagrożenia dla niepodważalności KSI, ale rozwój komputerów klasycznych (wydaje mi się, że) tak. Umiejętność znalezienia danych, które dają określoną wartość funkcji skrótu jest zagrożeniem dla KSI.

Szybsze komputery będą mogły obliczać więcej wartości funkcji skrótu w jednostce czasu i przez to bardziej efektywnie szukać danych, które dają określoną wartość funkcji skrótu.

Być może w przyszłości zostaną odkryte słabości stosowanych funkcji skrótu co ułatwi ich odwracanie.